articles/native-platforms/ios-objc/00-introduction.md

```diff
@@ -0,0 +1,160 @@
+
+<%= include('../../_includes/_github', { link: 'https://github.com/auth0-samples/auth0-ios-objc-sample/tree/master/00-Starter-Seed', }) %>
+
+This seed project can be useful as a starting point for the app to which you need to integrate authentication.
+
+
+
+## Sample Projects
+
+Each tutorial in the serie includes a link to its corresponding sample project, which expose how to achieve the goal from the tutorial in question. You can check out all the samples [here](https://github.com/auth0-samples/auth0-ios-objc-sample/).
+
+
+
+## Dependencies
+
+Each tutorial will require you to use either [Lock](https://github.com/auth0/Lock.iOS-OSX) or the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit, or both. 
+
+Briefly speaking about them:
+
+- [**Lock**](https://github.com/auth0/Lock.iOS-OSX) is a widget that is easy to present in your app. It contains default templates (that can be a bit customized) for login with email/password, sign up, social providers integration, and also password recovery.
+- [**Auth0.swift**](https://github.com/auth0/Auth0.swift) is a toolkit that lets you communicate with many of the basic [Auth0 API](https://auth0.com/docs/api) functions in a neat way.
+
+These dependencies are already integrated on each sample project through [Cocoapods](https://cocoapods.org/). We have chosen to use an approach of not git-ignoring them in the samples repo; that way, you don't have to worry about running pod commands: once you've cloned the repo, the samples are ready to be run!
+
+It's recommended that, as a starting point for your project, you download the [seed project](#seed-project) described above, which already includes these dependencies.
+
+Otherwise, if you're starting an empty project by your own, or if you want to add authentication to an existing project from yours, you have to make sure you install them:
+
+#### Carthage
+
+If you are using Carthage, add the following lines to your `Cartfile`:
+
+```ruby
+github "auth0/Lock.iOS-OSX" -> 1.26
+github "auth0/Auth0.swift" "1.0.0-beta.5"
+```
+
+Then, run `carthage bootstrap`.
+
+> For more information about Carthage usage, check [their official documentation](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos).
+
+#### Cocoapods
+
+If you are using [Cocoapods](https://cocoapods.org/), add these lines to your `Podfile`:
+
+```ruby
+use_frameworks!
+pod 'Lock', '~> 1.24'
+pod 'Auth0', '1.0.0-beta.5'
+```
+
+Then, run `pod install`.
+
+> For further reference on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
+
+
+
+## Your First Steps
+
+Make sure you complete these steps before going through any tutorial:
+
+#### 1. Create an application
+
+<%= include('../../_includes/_new_app') %>_
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+#### 2. Configure your callback URLs
+
+<%= include('../_includes/_callback-url-introduction') %>
```

Comment:
@sebacancinos @pablolvillar This does not exist `../_includes/_callback-url-introduction` (neither the file not the directory) and the page cannot load due to that.